style.css

```diff
@@ -95,9 +95,136 @@ a:hover {
   margin: 0 0 12px 24px;
 }
 
-ul {
+ul.social-icons {
   display: flex;
   justify-content: flex-start;
   align-items: center;
   gap: 30px;
 }
+
+.projects-section {
+  display: flex;
+  flex-direction: column;
```

Comment:
We are [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to use `CSS grid` to lay out the work section and `flexbox` for the contents in the section😉 You have made use of `flexbox` for its content, but the entire section should be built with `CSS grid`😉 Kindly use `grid` instead of `flex` here😎
<img width="793" alt="CSSGRUD ADB FKEX" src="https://user-images.githubusercontent.com/64643995/220664614-3c8b7544-92bd-4281-8ee5-3ea95b621c18.png">

Reply:
@Ibizugbe I used the css Grid to layout the work section as required